mods/ra/rules/world.yaml

```diff
@@ -4,6 +4,8 @@
 	ActorMap:
 	ScreenMap:
 	MusicPlaylist:
+		StartingMusic: intro
```

Comment:
I guess that this is in transit.mix, because I don't have it.  Some of our content mirrors don't include transit.mix (but some do, IIRC!), so we're going to need to fix that and then add it to our content file check in mod.yaml.